content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md

```diff
@@ -0,0 +1,101 @@
+
+For more architecture detail concerning PLDM, read the [Overview of the Platform Level Data Model for Redfish Device Enablement Standard](https://developer.hpe.com/blog/overview-of-the-platform-level-data-model-for-redfish%C2%AE-device-enablement-standard/) (PLDM for RDE) blog post.
+
+## Classification of the different firmware update types
+
+Computer firmware update is a complex process involving several types of objects in different contexts. An attempt to explain how the Redfish service of HPE iLO implements this process is presented in the following three blog posts:
+
+* [HPE firmware updates: Part 1 - File types and Smart Components](https://developer.hpe.com/blog/hpe-firmware-updates-part-3-the-redfish-update-service/)
+* [HPE firmware updates: Part 2 - Interaction in operating modes](https://developer.hpe.com/blog/hpe-firmware-updates-part-2-interaction-in-operating-modes/)
+* [HPE firmware updates: Part 3 - The Redfish update service](https://developer.hpe.com/blog/hpe-firmware-updates-part-3-the-redfish-update-service/)
+
+Although the above articles provide many definitions, acronyms and their relationship with the firmware update process, I will use, in this blog post, a different approach to this process than the one used in the three articles mentioned above.
+
+Firmware updates can be classified in four types called A, B, C and D. This classification is HPE specific but you may read or hear references to it, like in the [HPE Redfish API Reference Document](https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/#configuration).
+
+### Type A firmware update
+
+In this type of firmware update, the firmware can be flashed to its destination component by the iLO and become active without any server reboot required.
+
+Examples of firmware falling in this category:
+
+* iLO firmware
+* Power Supply firmware
+* PLDM firmware
+
+> Note: The last bullet of this list is the reason of this article.
+
+### Type B
+
+Firmware update of type B is similar to type A, but requires a host server reset for the firmware to become active.
+
+Examples of type B firmware:
+
+* Unified Extensible Firmware Interface (UEFI) firmware
+* Complex Programable Logic Devices (CPLD) firmware
+
+### Type C
+
+This firmware type contains firmware flashed by the UEFI. A system reboot is required for the new firmware to become active.
+
+Examples:
+
+* Trusted Platform Module (TPM) firmware
+* Non Volatile DIMM firmware
+
+### Type D
+
+UEFI and run time agents, like the Smart Update Manager (SUM) or the Smart Update Tool (SUT)) can flash the firmware content. This type of firmware update may require a system restart.
+
+Examples:
+
+* Smart Array controller firmware
+* Network Interface Cards (NICs) firmware
+
+> Note: devices depending associated to a type D firmware update are called *type-D devices*.
```

Comment:
I don't understand what this note is saying. "Devices depending..." on what?

Reply:
Removed word "depending"